.gitlab-ci.yml

```diff
@@ -1027,6 +1024,31 @@ workflow:
   - when: manual
     allow_failure: true
 
+.on_windows_systemprobe_or_e2e_changes:
+  - !reference [.on_e2e_main_release_or_rc]
+  - changes:
+      paths:
+        - pkg/collector/corechecks/ebpf/**/*
+        - pkg/collector/corechecks/servicediscovery/module/*
+        - pkg/ebpf/**/*
```

Comment:
eBPF is specific to Linux, so I'm not sure the following paths make sense here:
- `pkg/collector/corechecks/ebpf/**/*` (on_windows_systemprobe_or_e2e_changes)
- `pkg/ebpf/**/*` (on_windows_systemprobe_or_e2e_changes and on_windows_security_or_e2e_changes)

Reply:
Hmm interesting,
Oh yeah indeed the code is not even present when we build it on Windows. I will remove that path then!
Thanks for the comment